edx_toggles/urls.py

```diff
@@ -8,5 +8,5 @@
 from django.views.generic import TemplateView
 
 urlpatterns = [
-    url(r'', TemplateView.as_view(template_name="edx_toggles/base.html")),
+    url('', TemplateView.as_view(template_name="edx_toggles/base.html")),
```

Comment:
I believe you're missing this template, and maybe it should be in the `templates` folder?

Reply:
it exists here: https://github.com/edx/edx-toggles/blob/master/edx_toggles/templates/edx_toggles/base.html
(posting here to save this discussion to history)

Reply:
the templates folder in parent directory was for templates for toggles report. I’m deleting the folder now